pyproject.toml

```diff
@@ -24,23 +24,26 @@ classifiers = [
     'Topic :: Scientific/Engineering :: Artificial Intelligence',
 ]
 dependencies = [
-    "numpy >=1.21.0",
+    "numpy>=1.21.0",
     "h5py>=3.8.0",
     "tqdm>=4.65.0",
     "typing_extensions>=4.4.0",
-    "google-cloud-storage==2.5.0",
-    "typer[all]==0.9.0",
+    "google-cloud-storage>=2.5.0",
+    "typer[all]>=0.9.0",
     "gymnasium>=0.28.1",
-    "portion==2.4.0",
-    "packaging==23.1",
+    "portion>=2.4.0",
+    "packaging>=23.1",
 ]
 dynamic = ["version"]
 
 [project.optional-dependencies]
 testing = [
-    "pytest ==7.1.3",
-    "gymnasium-robotics>=1.2.1",
-    "imageio >=2.14.1"
+    "pytest>=7.1.3",
+    "pytest-markdown-docs>=0.4.3",
+    "nbmake",
+    "gymnasium-robotics>=1.2.3",
+    "imageio>=2.14.1",
+    "minigrid",
```

Comment:
Should minigrid be versioned? Probably doesn’t matter too much but maybe good to set a minimum